guides/manipulating-pages.rst

```diff
@@ -1,102 +1,96 @@
 Manipulating Pages
 ==================
 
-Ok, you've got an interesting page element. Now you'll want to do something
-with it. ``Behat\Mink\Element\NodeElement`` provides bunch a of useful methods
-for you:
+Once you :doc:`get a page element </guides/traversing-pages>`, you will want
+to manipulate it. You can also interact with the page, which is covered in
+the :doc:`next chapter </guides/interacting-with-pages>`.
+
+Getting the tag name
+--------------------
+
+The ``NodeElement::getTagName`` method allows you to get the tag name of
+the element. This tag name is always returned lowercased.
 
 .. code-block:: php
 
     $el = $page->find('css', '.something');
 
     // get tag name:
-    echo $el->getTagName();
+    echo $el->getTagName(); // displays 'a'
 
-    // check that element has href attribute:
-    $el->hasAttribute('href');
+Accessing HTML attributes
+-------------------------
 
-    // get element's href attribute:
-    echo $el->getAttribute('href');
-
-Element Content and Text
-~~~~~~~~~~~~~~~~~~~~~~~~
-
-To retrieve HTML content or plain text from out of the element, you can use:
-
-.. code-block:: php
+The ``NodeElement`` class gives you access to HTML attributes of the element.
 
-    $plainText = $el->getText();
-    $html = $el->getHtml();
-
-.. note::
-
-    ``getText()`` will strip tags and unprinted characters out of the response,
-    including newlines. So it'll basically return the text, that user sees
-    on the page.
+``NodeElement::hasAttribute``
+    Checks whether the element has a given attribute.
 
-Form Field Manipulations
-~~~~~~~~~~~~~~~~~~~~~~~~
+``NodeElement::getAttribute``
+    Gets the value of an attribute.
 
-You can fill form fields/retrieve its values with form manipulation actions:
+``NodeElement::hasClass``
+    Checks whether the element has the given class (convenience wrapper around
```

Comment:
well, this is why it is a convenience wrapper: checking that an element has a class is not just checking that it has a class attribute, but also splitting it as a space-separately list.